lib/tests/streamlit/runtime/caching/cache_data_api_test.py

```diff
@@ -661,6 +661,11 @@ def test_shows_cached_widget_replay_warning(
     ):
         """Test that a warning is shown when a widget is created inside a cached function."""
 
+        if _widget_name == "experimental_audio_input":
```

Comment:
Here and in `cache_resource_api_test.py` we explicitly exclude `experimental_audio_input` because it is sending two messages, one for deprecation warning alert, and another for actual `audio_input`